crates/holochain/src/core/ribosome/real_ribosome/mod.rs

```diff
@@ -929,9 +924,7 @@ impl RibosomeT for RealRibosome {
                 match zome.zome_def() {
                     ZomeDef::Wasm(wasm_zome) => {
                         let module = if let Some(path) = wasm_zome.preserialized_path.as_ref() {
-                            Arc::new(holochain_wasmer_host::module::get_ios_module_from_file(
-                                path,
-                            )?)
+                            self.preserialized_module(path)?
```

Comment:
Hmm yeah I guess it would if you had the `dylib` key in your dna manifest. That's not right -- I'll update to ignore the preserialization_path in wamr mode.

Reply:
Actually it should be able to deserialize in wamr mode, just not compile+serialize. I'll dig in more.

Reply:
So even though it could be deserialized in wamr mode, it doesn't make sense to as it would just be run through the interpreter anyway.

I just updated the implementation to only use the preserialized module in sys mode -- let me know what you think. I'm trying to keep all the feature flag gated code in separate files, but not sure if that makes it clearer or less clear.